lib/internal/modules/esm/resolve.js

```diff
@@ -46,7 +47,11 @@ const {
   ERR_UNSUPPORTED_ESM_URL_SCHEME,
 } = require('internal/errors').codes;
 
-const DEFAULT_CONDITIONS = ObjectFreeze(['node', 'import']);
+const DEFAULT_CONDITIONS = ObjectFreeze([
```

Comment:
q: wondering if this array is used in an order-sensitive manner? if yes - which of those conditions has the top priority? is it `"development"` or `"node"`?

Reply:
Order used to matter, but it was deemed too magical - instead the order in the object in the package.json applies so this array order is irrelevant.

Reply:
k, makes sense - thanks for the clarification 👍